docs/api/en/materials/Material.html

```diff
@@ -292,7 +292,7 @@ <h3>[property:String uuid]</h3>
 
 		<h3>[property:Integer version]</h3>
 		<p>
-		This starts at `0` and counts how many times [property:Boolean needsUpdate] is set to `true`.
+		This starts at `0` and counts how many times [page:Material.needsUpdate .needsUpdate] is set to `true`.
```

Comment:
Not sure this change is necessary. The links also work with the existing code.

Reply:
![image](https://user-images.githubusercontent.com/92771507/185384549-bc715d5e-9183-44f0-91db-7589b83309db.png)
The current site shows a `: Boolean` after the `.needsUpdate` link, which is inconsistent and confusing due to the weird spacing.